src/Pages/Top.elm

```diff
@@ -164,4 +169,41 @@ viewManifestList manifests =
 
 viewManifest : Manifest -> Element Msg
 viewManifest manifest =
-    row [] [ text manifest.shortName ]
+    row [ spacing 20
+        , padding 10
+        , width fill
+        ]
+        [ column []
+            [ image [ width (Element.px 24)]
+                { src = "../public/images/badge-outline-colored.svg"
+                , description = "App Icon goes here"
+                } ]
+        , column [ width fill ]
+            [ link
+                []
+                { url = Route.toString (Route.Preview__AppShortName_String {appShortName = manifest.shortName })
+                , label = text manifest.name
+                }
+            ]
+        , column [ Background.color (viewColor manifest.backgroundColor)
```

Comment:
The color boxes are not visible when the color is white. Could we add a light gray border around the boxes to bring them out?

Reply:
good point! added